USFMToolsSharp/Models/Markers/FKMarker.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace USFMToolsSharp.Models.Markers
+{
+    /// <summary>
+    /// Footnote keyword Marker
+    /// </summary>
+    class FKMarker : Marker
+    {
+        public override string Identifier => "fk";
+        public string FootNoteKeyword;
+
+
+        public override string PreProcess(string input)
+        {
+            FootNoteKeyword = input.ToUpper();

```

Comment:
This is incorrect. Formatting is the responsibility of the renderers.

Reply:
Moved ToUpper() to HTMLRenderer.